schema/pet-insurance.json

```diff
@@ -461,45 +1216,45 @@
                         },
                         "firstName": {
                             "type": "string",
-                            "description": "",
-                            "example": "Jon"
+                            "description": "First name(s) or given name(s), as specified in ID.",
+                            "minLength": 1,
+                            "example": "Bob"
                         },
                         "middleName": {
                             "type": "string",
                             "default": "",
-                            "description": ""
+                            "description": "Middle name, if applicable.<br/>This will only be used on websites which provide separate entry for middle names, otherwise it will be ignored.<br/>If middle name is essential for placing order, consider appending it to <code>firstName</code>."
                         },
                         "lastName": {
                             "type": "string",
-                            "description": "",
-                            "example": "Doe"
+                            "description": "Last name or surname, as specified in ID.",
+                            "minLength": 1,
+                            "example": "Smith"
                         },
                         "dateOfBirth": {
                             "type": "string",
-                            "description": "Date of birth in YYYY-MM-DD format.",
+                            "description": "Proposer's date of birth in YYYY-MM-DD format.",
                             "format": "date",
                             "example": "1976-01-27"
                         }
-                    }
+                    },
+                    "required": [
+                        "title",
+                        "firstName",
+                        "lastName",
+                        "dateOfBirth"
+                    ]
                 },
                 "address": {
-                    "property": {
-                        "number": {
-                            "type": "string",
-                            "description": "",
-                            "example": "123"
-                        },
-                        "name": {
-                            "type": "string",
-                            "default": "",
-                            "description": ""
-                        }
-                    },
                     "postcode": {
                         "type": "string",
-                        "description": "",
-                        "example": "W1W 5BB"
-                    }
+                        "description": "Postcode in country-specific format, e.g. 5-digit number in US or <code>E3 3RP</code> in UK.",
+                        "minLength": 1,
+                        "example": "94581"

```

Comment:
Since everything in here is restricted to GB, let's add British examples.

Reply:
I'll fix.